docs/website/docs/dlt-ecosystem/verified-sources/rest_api.md

```diff
@@ -608,28 +609,56 @@ When the API endpoint supports incremental loading, you can configure the source
         "since": {
             "type": "incremental",
             "cursor_path": "updated_at",
-            "initial_value": "2024-01-25T11:21:28Z",
+            "initial_value": "2024-01-25T00:00:00Z",
```

Comment:
I changed it into zeroes so that it is clearer when we show the url-encoded query string below